src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -416,6 +416,60 @@ private Locomotive performSwipe(SwipeElementDirection direction, WebElement elem
         return this;
     }
 
+
+
+
+    public WebElement swipeTo(SwipeElementDirection s, By by, int attempts){
```

Comment:
Might be good to create a few overload methods for this
- `public WebElement swipeTo(By by)`
  - `SwipeDirection` could default to `UP` (scrolling down on the screen)
  - `attempts` has a default value of X (pick a number??)
- `public WebElement swipeTo(By by, SwipeDirection direction)`
- `public WebElement swipeTo(By by, int attempts)`

They all overload to this method.

Reply:
I forgot the overloads for the `String id`s

Reply:
ah good thinking! done